tests/PHPUnit/Integration/EmailValidatorTest.php

```diff
@@ -8,6 +8,9 @@
 
 namespace Piwik\Tests\Integration;
 
+use Guzzle\Http\Client;
+use Guzzle\Http\Message\Request;
+use Guzzle\Http\Message\Response;
```

Comment:
FYI: Will guzzle is a library in Piwik it is only their for AWS-SDK. It's not a supported lib and we might remove it at some point. Instead use `Piwik\Http`.